tests/test_app/TestApp/Controller/DependenciesController.php

```diff
@@ -47,7 +47,7 @@ public function variadic()
         return $this->response->withStringBody(json_encode(['args' => func_get_args()]));
     }
 
-    public function spread(...$args)
+    public function spread(string ...$args)
```

Comment:
Was this receiving null and converting to an empty string somewhere passing the test?

Reply:
It was getting a null, which was silently cast to an empty array by the spread operator.